phoebe/backend/interferometry.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python3
+
+"""
+interferometry.py
+
+Interferometric module for Phoebe.
+
+Reference: Hanbury-Brown et al. (1974, MNRAS 167, 121-136).
+Reference: Brož (2017, ApJS 230, 19).
+
+"""
+
+import sys
```

Comment:
This is only used in a commented debug line.  If those commented lines should stay, then maybe we can move the import into that block?

```suggestion
```

Reply:
Corrected.